03_noise.ipynb

```diff
@@ -280,7 +281,7 @@
     "            )\n",
     "        )\n",
     "    )\n",
-    "    output_map /= np.sqrt(get_tube_years(config, site, channel))\n",
+    "    output_map /= np.sqrt(get_tube_years(config, site, channel)/simulated_tubes[band])\n",
```

Comment:
@keskitalo please doublecheck here:

* `simulated_tubes` is defined above in `00_core`
* if we simulate SAT, this is 1 so nothing changes
* if we simulate e.g. Chile LAT MFL where we have 54 tubes for each telescope, tube-years is 54*2*7, but the simulated tubes are 54, so I scale down the sims by a factor of sqrt(14)
.

Reply:
Yes. Seems correct.